docs/source/conf.py

```diff
@@ -56,6 +57,15 @@
 # The suffix(es) of source filenames.
 source_suffix = [".md"]
 
+# -- MyST configuration ------------------------------------------------------
+# ref: https://myst-parser.readthedocs.io/en/latest/configuration.html
+#
+
+myst_enable_extensions = [
+    # available extensions: https://myst-parser.readthedocs.io/en/latest/syntax/optional.html
+    "colon_fence",
+]
+
```

Comment:
I believe this a leftover from https://github.com/jupyterhub/pytest-jupyterhub/pull/44/commits/8133acc64ee715c260739d3b1267bac072bd099a and it's not actually used.